packages/x-codemod/src/v6.0.0/pickers/rename-components-to-slots/index.ts

```diff
@@ -0,0 +1,71 @@
+function transformComponentsProp(attributeNode) {
+  attributeNode.name.name = 'slots';
+
+  const valueExpression = attributeNode.value.expression;
+  if (valueExpression?.type !== 'ObjectExpression') {
+    return;
+  }
+
+  valueExpression.properties.forEach((property) => {
+    property.key.name = property.key.name[0].toLowerCase() + property.key.name.slice(1);
+
+    if (property.shorthand) {
+      property.shorthand = false;
+    }
+  });
+}
+
+function transformComponentsPropsProp(attributeNode) {
+  attributeNode.name.name = 'slotProps';
+}
+
+/**
+ * @param {import('jscodeshift').FileInfo} file
+ * @param {import('jscodeshift').API} api
+ */
+export default function transformer(file, api, options) {
+  const j = api.jscodeshift;
+
+  const printOptions = options.printOptions;
+
+  const root = j(file.source);
+
+  root
+    .find(j.ImportDeclaration)
+    .filter(({ node }) => {
+      return node.source.value.startsWith('@mui/x-date-picker');
```

Comment:
Nitpick: Maybe this codemod could not be `pickers` specific but handle both data grid as well as pickers changes?
I believe that both packages did make the change of deprecating the old `components` naming and adding `slots`. 🤔

Reply:
It would not make sense with the comande line name
```sh
npx @mui/x-codemod v6.0.0/pickers/migrate-to-components-componentsProps
```

Reply:
Of course, that's why I was thinking if it wouldn't make sense to just move this codemod to top level? 🤔

Reply:
Or may be create a util and reuse in both data grid and pickers

Reply:
Yes, that would allow users to do the migration in two steps: the data grid then the pickers (or the opposit)